easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '3.9.0'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+checksums = ['66c73179da42cee1386371641241f79ded250e117a79f571bbd69e56daa48948']
+
+builddependencies = [
+    ('CMake', '3.7.1'),
+    ('Python', '2.7.12', '-bare'),
+]
+
+dependencies = [
+    ('ncurses', '6.0'),
```

Comment:
`CMake` 3.7.1 uses `ncurses` 6.0 [1]. Why should LLVM link against 5.9?

[1] https://github.com/easybuilders/easybuild-easyconfigs/blob/fabcff68ef979dc437da5fdecd671d99c69402cf/easybuild/easyconfigs/c/CMake/CMake-3.7.1-GCCcore-5.4.0.eb